### PR TITLE
Update FileInput styling

### PR DESCRIPTION
**Pull request by lukenewman**
## Background
Add styling to the `FileInput` component to support redesign of the initial upload page.

## Associated PRs
incoming

### CODE REVIEW

**Review by vonbearshark — APPROVED** [review]

**Review by JordanGuinn — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ctoec/component-library",
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (jessethach) [inline package.json:3]: Is this a minor or patch change? If it's a patch, I think we can increment to `2.0.1`

**Review by jessethach — APPROVED** [review]
Just a comment about version bump!

File: package.json (hunk 1 above)
Comment (lukenewman) [reply package.json:3]: it's a design update, so i was thinking of bumping the minor, but open to whatever. are patches reserved for bug fixes?

Comment (jessethach) [reply package.json:3]: Ah gotcha - thought this was a bug fix. I think you're right!

**Pull request closed without merging**